cognite/client/data_classes/data_modeling/debug.py

```diff
@@ -0,0 +1,475 @@
+from __future__ import annotations
+
+from abc import ABC
+from dataclasses import dataclass
+from functools import cache
+from typing import TYPE_CHECKING, Any, Literal, NoReturn, cast, get_args, get_type_hints
+
+from typing_extensions import Self
+
+from cognite.client.data_classes._base import CogniteResource, CogniteResourceList
+from cognite.client.data_classes.data_modeling.ids import ContainerId
+from cognite.client.data_classes.data_modeling.instances import InstanceSort
+from cognite.client.utils._auxiliary import all_concrete_subclasses
+
+if TYPE_CHECKING:
+    from cognite.client import CogniteClient
+
+
+@dataclass(kw_only=True)
+class DebugParameters:
+    """
+    Debug parameters for debugging and analyzing queries.
+
+    Args:
+        emit_results (bool): Include the query result in the response. emit_results=False is required for advanced query analysis features.
+        timeout (int | None): Query timeout in milliseconds. Can be used to override the default timeout when analysing queries. Requires emit_results=False.
+        profile (bool): Most thorough level of query analysis. Requires emit_results=False.
+    """
+
+    emit_results: bool = True
+    timeout: int | None = None
+    profile: bool = False
```

Comment:
The api docs contains a few more parameters here, `includeTranslatedQuery` and `includePlan`. Are these excluded on purpose?

Reply:
They are in alpha. Do you think we should add them right away? 🤔

Reply:
If you're aware, and it's intentional, I have no objections.